CMakeLists.txt

```diff
@@ -104,11 +104,7 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake ${ROCM_PATH}/lib
 list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/lib/cmake/hip /opt/rocm /opt/rocm/llvm /opt/rocm/hip)
 
 option(BUILD_SHARED_LIBS "Build hipSOLVER as a shared library" ON)
-if(BUILD_SHARED_LIBS)
-  set(BUILD_WITH_SPARSE_DEFAULT OFF)
-else()
-  set(BUILD_WITH_SPARSE_DEFAULT ON)
-endif()
+set(BUILD_WITH_SPARSE_DEFAULT OFF)
```

Comment:
You can probably remove BUILD_WITH_SPARSE_DEFAULT entirely. I think we only added it to manage the different defaults between the shared and static lib.

Also, do you not need to modify load_function, load_rocsparse, and load_cholmod the same way as in rocSOLVER?

Reply:
We ought to remove it from line 112 as well. I'll approve once that's done.